content/en/containers/guide/operator-eks-addon.md

```diff
@@ -0,0 +1,152 @@
+   ```
+   Replace `<DATADOG_API_KEY>` and `<DATADOG_APP_KEY>` with your [Datadog API and application keys][5].
+
+
+3. Create a `datadog-agent.yaml` file with the spec of your `DatadogAgent` deployment configuration. The Datadog Operator uses default Agent and Cluster Agent image settings and pulls them from a public registry.
+
+   If you want to pull images from a private EKS registry, you can add `global.registry`. The following configuration enables metrics, logs, and APM:
+   ```yaml
+   apiVersion: datadoghq.com/v2alpha1
+   kind: DatadogAgent
+   metadata:
+     name: datadog
+   spec:
+     global:
+       # required in case Agent can not resolve cluster name through IMDS, see the note below.
+       clusterName: <CLUSTER_NAME>
+       registry: 709825985650.dkr.ecr.us-east-1.amazonaws.com/datadog
+       credentials:
+         apiSecret:
+           secretName: datadog-secret
+           keyName: api-key
+         appSecret:
+           secretName: datadog-secret
+           keyName: app-key
+     features:
+       apm:
+         enabled: true
+       logCollection:
+         enabled: true
+   ```
+   This agent instance configuration pulls the Datadog agent image from an AWS Marketplace hosted ECR repository which also contains the image for the Datadog Operator Amazon EKS add-on. Should you require alternatives, kindly edit the 'global.registry' entry in the manifest above.
+
+   For all configuration options, see the [Operator configuration spec][6].
+
+   **Note:** If access to IMDS v1 is blocked on the node, Agent will not be able to resolve the cluster name and certain features for example [Orchestrator Explorer][6] will not work. Hence, it is recommended to add `spec.global.ClusterName` in the `DatadogAgent` manifest. See this [comment][8] on how to configure Agent to request metadata using IMDS v2.
+
+4. Deploy the Datadog Agent:
+   ```bash
+   kubectl apply -f /path/to/your/datadog-agent.yaml
+   ```
+
+
+## Uninstalling Operator
+
+If you want to uninstall the Agent and Operator, first delete the `DatadogAgent` custom resource:
+
+  ```bash
+  kubectl delete datadogagents.datadoghq.com datadog
+  ```
+
+Confirm all Agent resources are deleted and proceed with add-on uninstallation:
+
+{{< tabs >}}
+{{% tab "Console" %}}
+
+* Go to the EKS cluster in the AWS console.
+* Go to the add-on tab and select *Datadog Operator* addon.
+* Click Remove and the confirmation prompt.
+
+{{% /tab %}}
+{{% tab "CLI" %}}
+
+To delete the add-on, run:
+  ```bash
+  aws eks delete-addon --addon-name datadog_operator --region <AWS_REGION> --cluster-name <CLUSTER_NAME>
+  ```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+ **Note:** If Operator add-on is uninstalled before deleting the `DatadogAgent` custom resource, agents will continue to run on the cluster. Deleting namespace will get stuck since the `DatadogAgent` can not be finalized without a running Operator. See this Github [issue][9] for a workaround.
```

Comment:
```suggestion
 **Note:** If you uninstall the Operator add-on before deleting the `DatadogAgent` custom resource, Agents continue to run on the cluster. Deleting the namespace fails since the `DatadogAgent` cannot be finalized without a running Operator. See this Github [issue][9] for a workaround.
```